### DISCUSSION TITLE
LaunchPoint Draft Ideas

### DISCUSSION THREAD

**Discussion by AdamW19** [AdamW19/LaunchBot#32] [Ideas]
We're soliciting ideas on how to better implement the draft.

The main end goal is to automate draft match results, have a leaderboard, and to implement a player ranking system.

We have a few ideas from LaunchPoint members that boils down to 2 main ideas:

1. Have players make their own lobby and have LaunchBot form an embed lobby for map-mode randomization and such.

2. Have players finish the draft set, and report the scores in a dedicated channel with a set reporting format to the bot to parse and process.

If you have any ideas/thoughts/comments, feel free to let us know here!

**Comment by GuadarramaDelAngel-Jorge** [comment]
Hello Adam! This is Flux.
IMO, I think y'all can split the idea in 2 or 3 parts, depending on how you see it.
1. I think the most important and necessary feature is the automation of draft match results. (Maybe release 1.0?)
Fix bugs and everything, bla bla bla (Versions 1.1....)
2. Automation of the creation of the draft, get the maps and modes and prevent duplicates, etc. (2.0)
Fix bugs, etc (2.1.....)
3. Focus on the leaderboard and ranking system which I think it's the most complex part that can be isolated.

Thanks for your attention!
